Testing/ORKTest/ORKTestTests/ORKJSONSerializationTests.m

```diff
@@ -569,13 +577,30 @@ - (void)testORKSecureCoding {
         }
         
         NSData *data2 = [NSKeyedArchiver archivedDataWithRootObject:newInstance];
+        
+        NSKeyedUnarchiver *unarchiver2 = [[NSKeyedUnarchiver alloc] initForReadingWithData:data2];
+        unarchiver2.requiresSecureCoding = YES;
+        unarchiver2.delegate = self;
+        id newInstance2 = [unarchiver2 decodeObjectOfClasses:[NSSet setWithArray:classesWithSecureCoding] forKey:NSKeyedArchiveRootObjectKey];
+        NSData *data3 = [NSKeyedArchiver archivedDataWithRootObject:newInstance2];
+        
         if (![data isEqualToData:data2]) { // allow breakpointing
```

Comment:
This if check to allow breakpointing seems unneeded. One can use the "Test Failure" breakpoint instead.